### DISCUSSION TITLE
An alternative way to support Gromacs

### DISCUSSION THREAD

**Discussion by fhh2626** [fhh2626/BFEE2#80] [Ideas]
I think that we can support Gromacs using ParmEd. If so, many current functions can be reused. We only need to reimplement methods writing configurational files.

**Comment by HanatoK** [comment]
@fhh2626 As a GUI application, users often generate input files on their desktop computers, and then copy them to servers for running. The servers may or may not have available python installed. If there is no python intepreter then one cannot use Parmed on the server to prepare input files. Does your approach require Parmed and python on the server?
